### PR TITLE
docs: fix typo for `docs` folder

### PR DESCRIPTION
**Pull request by Hongarc**
Small fix typo.

Correct me if I'm wrong, thank you.

### CODE REVIEW

File: docs/getting-started-publishing.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ This will generate a `build` directory inside the `website` directory containing
 
 ## Hosting Static HTML Pages
 
-At this point, you can grab all of the files inside the `website/build` directory and copy them over to your favorite web server's `html` directory.
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: This should still be lower case.

File: docs/getting-started-publishing.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ Now, whenever a new commit lands in `master`, Travis CI will run your suite of t
 Steps to configure your Docusaurus-powered site on Netlify.
 
 1.  Select **New site from Git**
-1.  Connect to your preferred Git provider.
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Should still be lower case

File: docs/guides-navigation.md [hunk 3]
```diff
@@ -219,7 +219,7 @@ You have limited control where the search and languages dropdown elements are sh
 
 ### Search
 
-If search is enabled on your site, your search bar will appear to the right of your links. If you want to put the search bar between links in the header, add a search entry in the `headerLinks` config array:
```
Comment (yangshun) [inline docs/guides-navigation.md]: No need "the"

File: docs/guides-navigation.md [hunk 4]
```diff
@@ -234,7 +234,7 @@ If search is enabled on your site, your search bar will appear to the right of y
 
 ### Languages Dropdown
 
-If translations is enabled on your site, the language dropdown will appear to the right of your links (and to the left of the search bar, if search is enabled). If you want to put the language selection drop down between links in the header, add a languages entry in the `headerLinks` config array:
```
Comment (yangshun) [inline docs/guides-navigation.md]: No need "the" before search.

File: docs/guides-navigation.md [hunk 5]
```diff
@@ -271,7 +271,7 @@ We support secondary on-page navigation so you can more easily see the topics as
 }
 ```
 
-Currently, `'separate'` is the only option available for this field. This provides a separate navigation on the right side of the page.
```
Comment (yangshun) [inline docs/guides-navigation.md]: Keep "a" here

File: docs/guides-search.md [hunk 6]
```diff
@@ -3,9 +3,9 @@ id: search
 title: Enabling Search
 ---
 
-Docusaurus supports search using [Algolia DocSearch](https://community.algolia.com/docsearch/). Once your website is online, you can [submit it to DocSearch](https://community.algolia.com/docsearch/). Algolia will then send you credentials you can add to your `siteConfig.js`.
```
Comment (yangshun) [inline docs/guides-search.md]: "you" is correct.

File: docs/tutorial-create-new-site.md [hunk 7]
```diff
@@ -58,7 +58,7 @@ The following contents will be created for you in the directory you are in.
     └── yarn.lock
 ```
 
-There would be some example documentation and blog pages generated.
```
Comment (yangshun) [inline docs/tutorial-create-new-site.md]: No "of"

File: docs/tutorial-version.md [hunk 8]
```diff
@@ -46,12 +46,12 @@ If we go to the `localhost:3000/<baseUrl>/docs/doc1` URL in our browser, realize
 
 ## Next Version
 
-The latest versions of the docs have to be accessed by adding `next` to the URL: `localhost:3000/<baseUrl>/docs/next/doc1`. Note that the version beside the title also changes to `next` when we are on that URL.
```
Comment (yangshun) [inline docs/tutorial-version.md]: "beside" is right.

**Comment by Hongarc** [comment]
Ok, I removed them

**Pull request merged**